test.py

```diff
@@ -0,0 +1,3 @@
+print(1)
```

Comment:
What is test.py used for? Is it just for your debugging? In the future, please do explain all the changes you made to each file in your design.md

Reply:
Thank you for your comment. It is just for debugging and it has been removed. Sorry for the confusion!